src/TLuaInterpreter.cpp

```diff
@@ -4851,13 +4851,11 @@ int TLuaInterpreter::getLastLineNumber(lua_State* L)
 
 int TLuaInterpreter::getMudletHomeDir(lua_State* L)
 {
-    QString home = QDir::homePath();
-    home.append("/.config/mudlet/profiles/");
     Host& host = getHostFromLua(L);
-    QString name = host.getName();
-    home.append(name);
-    QString erg = QDir::toNativeSeparators(home);
-    lua_pushstring(L, erg.toLatin1().data());
+    QString nativeHomeDirectory = QDir::toNativeSeparators(mudlet::getMudletPath(mudlet::profileHomePath, host.getName()));
+    // Was using toLatin1() but that fails to accommodate Windows OS which allows
```

Comment:
Don't need a note here saying what it was using - we have git history if we want to dig back in time!

Reply:
It is just that it has been a long-standing issue for users like [__Angie/Alayla__]( https://forums.mudlet.org/help-forum-f9/getmudlethomedir-and-encoding-t4508.html#p20874) and it has been on my todo list for a couple of years now! :blush:

I'd be tempted to include a link to that forum page!